ansible/roles/bigip-ingress-setup/defaults/main.yml

```diff
@@ -0,0 +1,132 @@
+---
+# defaults file for bigip-ingress-setup
+
+########################
+## VM Input Variables ##
+########################
+
+bigip_vms:
+  bigip0:   # Don't change VM name
+    mgmt_ip: "192.168.223.100/24"
+    mgmt_gateway: "192.168.223.1"
+    int_sriov_ip: "192.168.222.100/24"   # Assign an IP from baremetal external CIDR
+    int_sriov_mac: "02:02:02:02:05:01"
+    ext_sriov_mac: "02:02:02:02:05:02"
+    license_key: "SGJVQ-MWRFS-UVBWU-NFKCX-NMDOIMF"
+    worker_node: "worker000-fc640"
+  bigip1:   # Don't change VM name
+    mgmt_ip: "192.168.223.101/24"
+    mgmt_gateway: "192.168.223.1"
+    int_sriov_ip: "192.168.222.101/24"   # Assign an IP from baremetal external CIDR
+    int_sriov_mac: "02:02:02:02:05:03"
+    ext_sriov_mac: "02:02:02:02:05:04"
+    license_key: "UVFIT-RVTVT-NAPUX-DPIVS-LNPEMYB"
+    worker_node: "worker001-fc640"
+
+bigip_bridges:
+  mgmt:
+    interface: "ens2f1"
+    vlan_id: "100"
+  ha:
+    interface: "ens2f1"
+    vlan_id: "101"
+
+bigip_sriov_int: "ens2f1"   # Preferrably external interface
+
+###################
+## IP Allocation ##
+###################
+
+bigip_sriov_network_ipam_address: "192.168.222.0/24"   # baremetal external CIDR
+bigip_sriov_network_ipam_gateway: "192.168.222.1"   # GW of baremetal external CIDR
+
+bigip_upstream_dns_snat_pool: ['192.168.222.160']     # Assign an IP from baremetal external CIDR
+bigip_upstream_dns: "192.168.222.1"   # External or Internal DNS 
+bigip_dns46_vs: "192.168.222.161"     # Assign an IP from baremetal external CIDR
+bigip_upstream_vs: "192.168.222.162"     # Assign an IP from baremetal external CIDR
```

Comment:
Lets take the extcidrnet and a range of IPs within it that can be used and then use jinja2 filters like https://github.com/redhat-performance/JetSki/blob/master/ansible-ipi-install/inventory/jetski/hosts#L116-L122 to assign IPs like this so that it's not a huge burden for the user.